src/ngraph/frontend/onnx_import/op/gemm.cpp

```diff
@@ -73,15 +71,22 @@ namespace ngraph
                     input_a = ngraph::builder::opset1::flatten(input_a, 1);
                     input_b = ngraph::builder::opset1::flatten(input_b, 1);
 
-                    auto matmul_node = std::make_shared<default_opset::MatMul>(input_a, input_b);
+                    std::shared_ptr<ngraph::Node> last_node =
```

Comment:
This code is now less clearly readable because of the changed variable names. Let's use the names we had before.
```suggestion
                    std::shared_ptr<ngraph::Node> matmul_node =
```

Reply:
Done.